lib/hackmode/index.js

```diff
@@ -244,9 +256,36 @@ class N3hHackMode extends N3hMode {
         if (!this._hasTrackOrFail(data.requesterAgentId, data.dnaAddress, data._id)) {
           return
         }
-        //  since we're fully connected, just redirect this back to itself for now...
-        data.method = 'handleFetchEntry'
-        this._ipcSend('json', data)
```

Comment:
can we switch these to `await` for readability?

Reply:
👍